README.md

```diff
@@ -326,8 +326,8 @@ Instead of using config in the HOC, you can put it in the Component class. This
 - touched: `object`
 - isValid: `boolean`
 - handleSubmit: `function`
-- handleChange: `(path || event, value) => {}`. event: `Event: {target: {name || id, value}} || CustomEvent: {detail: {name || id, value}}`
```

Comment:
need to be updated.

also add the Element.closest polyfill to the doc

Reply:
I knew that this is probably something that need some discussion so I left as is. WIP.